README.md

```diff
@@ -0,0 +1,154 @@
+	the fist the linear damping and the second quadratic damping.
+	It is based on [Smallwood][].
+
+	> **Note:**
+	For considering a decoupled model, one must set the matrices of inertia and damping as diagonal.
+
+
+ 2. COMPLEX:
+	It consider the Coriollis effect and a Damping Matrices with six terms,
+	all quadratic damping, been relative to the 3 linear and 3 angular DOF respectivelly.
+	It is based on [McFarland][].
+	
+ 3. INTERMEDIATE:
+	It consider the Coriollis effect and a Damping Matrices with two terms,
+	the first the linear and the second the quadratic damping.
+
+Other ways to formulate the model could be defined.
+
+### Inertia Matrix
+
+The $6 \times 6$ inertia matrix consider both the inertia of the vehicle and the added mass.
+
+$$ M = M_{vehicle} + M_{added\_mass} $$
+
+with, 
+
+$$ M_{vehicle} = 
+ \left( \begin{array}{cc}
+mass*Identity(3) & Zero(3) \\
+Zero(3) & Inertia \end{array} \right) 
+$$	
+
+### Damping Matrices
+
+The Damping Matrices is a vector of $6 \times 6$ matrices and his terms are defined according the Model Type.
+For the SIMPLE and INTERMEDIATE model type, the damping is represented by a linear and a quadratic term.
+
+$$ D = D_{linear}*v + D_{quadratic}*v*|v|$$
+
+For the COMPLEX case, the damping is represented by six quadratic terms, one for each DOF.
+
+$$ D = \left(\sum_{i=1}^{6}(|v_i|*D_i)\right) * v $$
+
+### Weight
+
+$$ Weight = Mass*gravity $$
+
+### Buoyancy
+
+Resultant buoyancy
+
+
+## Simulation <a id="simulation"></a>
+
+The simulation performs a integration of the states derivatives (acceleration and velocities)
+to the states of interest (velocities and position/orientation) using for that a 4th order Runge-Kutta.
+The parameters of the simulation are:
+
+ - Model Simulator
+ - Sampling Time 
+ - Simulation per Cycle
+ - Initial Time
+
+### Model Simulator
+
+Model Simulator defines if the integration of velocities to position/orientation will be made.
+The two possibilities are:
+
+ 1. DYNAMIC: Only the dynamic is take in account. It integrates the accelerations to update the velocities. 
+	For this case, updated orientation need to be set.
+
+ 2. DYNAMIC_KINEMATIC: It consider both the dynamic and the kinematic and do the intergration of all states.
+
+### Sampling Time
+ The step used in integration
+
+### Simulation per Cycle
+ Can be increased for precision purpose of the integration. 
+
+	
+[Fossen]:	https://scholar.google.com.br/citations?view_op=view_citation&hl=pt-BR&user=Sn8fzegAAAAJ&citation_for_view=Sn8fzegAAAAJ:u5HHmVD_uO8C
+[Smallwood]: http://ieeexplore.ieee.org/document/1208328/?arnumber=1208328&tag=1
+[McFarland]: http://ieeexplore.ieee.org/document/6631233/
+
+[^TODO]: Future improvement on the library may include the definition of an environment, with a floor (inferior limit) and a surface (superior limit) and the presence of water current. 
+It also may include the possibility to add more then one object and defines their interaction. 
```

Comment:
I like the idea of mentioning this possible improvements, so that we leave a flag raised to remind us that there is still work to be done here, but when the time comes, we need to discuss which of them are really meaningful for the library. ;)